hack/init-environment.sh

```diff
@@ -239,8 +241,8 @@ function install_cloudsql_proxy () {
     --version=0.0.5 \
     --set secrets[0].secretName=cloudsqlkey \
     --set secrets[0].vals[0].kubeSecretKey=cloudsqlkey.json \
-    --set secrets[0].vals[0].path=$vault_location \
-    --set secrets[0].vals[0].vaultKey=proxy_account_key
+    --set secrets[0].vals[0].path=vault_sa_location \
+    --set secrets[0].vals[0].vaultKey=sa_key
```

Comment:
Is "sa_key" an environment variable?

Reply:
I think it might be a string literal; "proxy_account_key" which was there before mapped to a key in the secret, so I followed that format. I'm not confident though

Reply:
Yes, it's a literal that needs to match the key in Vault